specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json

```diff
@@ -1746,7 +1746,7 @@
         },
         "enablePodSecurityPolicy": {
           "type": "boolean",
-          "description": "(PREVIEW) Whether to enable Kubernetes Pod security policy."
+          "description": "(DEPRECATING) Whether to enable Kubernetes Pod security policy."
```

Comment:
@JunSun17 could we actually update this to:

(DEPRECATING) Whether to enable Kubernetes pod security policy (preview). This feature is set for removal on October 15th, 2020. Learn more at aka.ms/aks/azpodpolicy.

Reply:
done.